src/test/java/com/checkmarx/ast/ScanTest.java

```diff
@@ -19,6 +23,22 @@ void testScanShow() throws Exception {
         Assertions.assertEquals(scanList.get(0).getId(), scan.getId());
     }
 
+    @Test
+    void testScanVorpalSuccessfulResponse() throws Exception {
```

Comment:
can we have a happy case too ?

Reply:
This is the happy case.. or you mean more assertions?